### PR TITLE
Adding deviceActor support

### PR DESCRIPTION
**Pull request by nicola**
This adds getDescription and getRawPermissionTable support.
However it doesn't for:
- screenshotToDataURL
- getWallpaper

since they both return longstr, which I am not totally aware how to handle yet.

### CODE REVIEW

File: lib/device.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+var extend = require("./extend"),
+    ClientMethods = require("./client-methods"),
+    Tab = require("./tab"),
+    fs = require("fs"),
```
Comment (harthur) [inline lib/device.js]: Shouldn't need `fs`, `Tab`, or `spawn`.

Comment (nicola) [inline lib/device.js]: Interesting, I wonder where I got them from!
Fixed in the newest commit, sorry for that.

**Comment by harthur** [comment]
Looks good, except for unnecessary requires. A test would be nice.

As for long string actors, you could create a new `LongString` class with methods `substring()`, and `getWallpaper()` could either return an instance of that class, or more likely substring the whole thing and return the plain string.

**Comment by harthur** [comment]
Merged

**Pull request closed without merging**